connectors/es/index.py

```diff
@@ -94,8 +92,6 @@ async def get_all_docs(self, query=None, sort=None, page_size=DEFAULT_PAGE_SIZE)
         Returns:
             Iterator
         """
-        await self.client.indices.refresh(index=self.index_name)
```

Comment:
good suggestion -- keeping refresh sounds useful to keep things a little bit faster. I'll find out

Reply:
We'll have an API at some point (see the slacks PMs), but until it's ready maybe we can just use a configuration flag?

Reply:
I'd suggest that maybe we don't actually need to refresh here, and it may be an anti-pattern to refresh as much as we're doing. I'm thinking of the issue we had where our frequent refreshes in the crawler were causing issues with fragmentation in the index. I think we should be evaluating a little more consciously whether a refresh is absolutely necessary, or whether it's fine to wait for the index to automatically refresh instead. In Kibana we've removed all refreshes, except for a single `refresh: 'wait_for'` when creating a connector (to make sure it shows up when redirecting the user).